ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/hub-side-templating/acm-example-multinode-site.yaml

```diff
@@ -0,0 +1,47 @@
+# For this PolicyGenerator, we've considered a multinode managed cluster with the following label(s):
```

Comment:
Better to refer here to the ConfigMap that will to be used with this template. Same apply for all templates.